<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StarWindow</class>
 <widget class="QWidget" name="StarWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StarWindow</string>
  </property>
  <widget class="QListWidget" name="gameListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>1000</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lifeListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>1001</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="sportListWidge">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>1000</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="readingListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>1001</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="travelListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>1000</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="scienceListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>1000</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="studyListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>1000</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="emailListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>1001</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="cartoonListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>1000</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="movieListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>1000</height>
    </rect>
   </property>
  </widget>
  <zorder>lifeListWidget</zorder>
  <zorder>sportListWidge</zorder>
  <zorder>readingListWidget</zorder>
  <zorder>travelListWidget</zorder>
  <zorder>scienceListWidget</zorder>
  <zorder>studyListWidget</zorder>
  <zorder>emailListWidget</zorder>
  <zorder>cartoonListWidget</zorder>
  <zorder>movieListWidget</zorder>
  <zorder>gameListWidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
